src/main/resources/META-INF/rewrite/ibm-java.yml

```diff
@@ -110,4 +112,30 @@ recipeList:
   - org.openrewrite.java.ChangeType:
       oldFullyQualifiedTypeName: com.ibm.security.auth.module.Krb5LoginModule
       newFullyQualifiedTypeName: com.sun.security.auth.module.Krb5LoginModule
-      ignoreDefinition: true
\ No newline at end of file
+      ignoreDefinition: true
+---
+type: specs.openrewrite.org/v1beta/recipe
+name: org.openrewrite.java.migrate.RemovedJavaXMLWSModuleProvided
+displayName: Do not use `java.xml.ws` module in application
```

Comment:
This display name is misleading for the recipe. We aren't saying the shouldn't use the package. We are suggesting that they provide the appropriate dependency. In the case of this recipe we will be removing any dependency they specify in their app since the appropriate dependency will be provided by the selected feature in their liberty server.xml